test_scripts/Defects/5_0/2454_SDL_does_not_clear_app_info_after_MASTER_RESET.lua

```diff
@@ -0,0 +1,62 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_core/issues/2454
+--
+-- Reproduction Steps:
+-- HMI and SDL are started.
+-- Perform MASTER_RESET
+
+-- Expected Behavior:
+-- All SDL data are cleaned and reset.
+---------------------------------------------------------------------------------------------------
+
+-- For useing this script you must set actual path to binary file directory of SDL to 'config.pathToSDL' in config.lua file.
```

Comment:
config.pathToSDL can be not defined in config.lua but defined as argument in terminal by script running.

Reply:
@GetmanetsIrina Changed in bf1c6b75fe5f4e1f3d891034a6faf412bcf250e1.